section3/exercises/arrays.js

```diff
@@ -106,11 +108,24 @@ var hasCharger = true;
 var chargerType = "car";
 
 // Write your conditional here
-
+// Lines 115-116 are setting up a conditional where there is also a logical operator used. The conditions that
+// these lines are setting up is: If the variable percentBatteryLeft is greater that or equal to 15
+// OR the variable hasCharger is equal to true AND chargerTypeis equal to "car",
+// then console log "You can call an Uber!"
+if ((percentBatteryLeft >= 15) || ((hasCharger == true) && (chargerType == "car"))) {
```

Comment:
Play around with which parenthesis are necessary in this if/else statement. We generally want to omit any that aren't needed.